### DISCUSSION TITLE
editable jusernames or note tabs

### DISCUSSION THREAD

**Discussion by DrLostGame** [SevenTV/chatterino7#202] [Ideas]
i would rly like to have the posibillity to edit the names of fellow chatters so i direktly see the notices i want for the user or some tab wher i can enter notices to look after things i noticed

**Comment by Nerixyz** [comment]
> to edit the names of fellow chatters

You can already do that. You can set a nickname for a user in the settings under **Nicknames**.

> so i direktly see the notices i want for the user or some tab wher i can enter notices to look after things i noticed

You can't add notes to users yet. This is tracked upstream in Chatterino/Chatterino2#2413.

_As this is a feature for upstream Chatterino, I'll close this discussion. Feel free to outline your use-case there._

**Reply by DrLostGame** [reply]
thank you for the help
